File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.py
def solution(n,a,b):
    answer = 1
    front = min(a,b)
    back = max(a,b)
    while True:
        if back - front == 1 and front %2 == 1:
            break
            
        front = front//2 + front%2
        back = back//2 +back%2
        answer += 1
    

    return answer

File: 프로그래머스/2/12985. 예상 대진표/예상 대진표.cpp
#include <iostream>

using namespace std;

int solution(int n, int a, int b)
{
    int answer = 0;
    
    int A = a;
    int B = b;
    while(1){
        if(A==B) break;
        A = (A/2) + (A%2);
        B = (B/2) + (B%2);
        answer++;
    }
    
    return answer;
}